waterlog/src/Globals.js

```diff
@@ -1,3 +1,3 @@
 export const Globals = {
-	API_URL: 'http://api.iot.retrotest.co.za'
+	API_URL: 'https://localhost:44382'
```

Comment:
are you supposed to be pointing locally?

Reply:
Sorry forgot to change it, now fixed.